CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 # [4.82.1]
```

Comment:
```suggestion
# [unreleased]
```

Reply:
Super! Here is one convention that you will not encounter so often:
when a new release has just happened, it may be that your PR us first in line to add a new CHANGELOG heading for the next new "unreleased" release. We used to do this as part of the version bump process, but saved in on the effort by just doing it for the first PR that goes to main after a release.

Reply:
It should be `## [unreleased]`, with 2 #.

I introduced an error in my previous merged PR, sorry! 😄

Reply:
I see, I'll update!